src/Exceptionless.Web/ClientApp/src/lib/features/shared/components/faceted-filter/faceted-filter-drop-down.svelte

```diff
@@ -92,9 +92,7 @@
     </Popover.Trigger>
     <Popover.Content align="start" class="p-0" side="bottom">
         <Command.Root {filter}>
-            {#if options.length > 10}
-                <Command.Input placeholder={title} />
-            {/if}
+            <Command.Input placeholder={title} autofocus={open} />
```

Comment:
if you conditionally show this, then usability goes way down because focus is all kinds of messed up and you can't tab between the components.